2.MLP_Multi_Classifier/MLP_utils/utils.py

```diff
@@ -0,0 +1,938 @@
+"""
+Functions for Machine Learning Model optimization, training and testing.
+These are helper functions meant to be called in a separate notebook or script
+"""
+
+
+import ast
+from configparser import ConfigParser
+from typing import Tuple
+
+import matplotlib.pyplot as plt
+import numpy as np
+import optuna
+import pandas as pd
+import seaborn as sns
+import torch
+import torch.nn as nn
+import torch.optim as optim
+from MLP_utils.parameters import Parameters
+from sklearn.metrics import (
+    accuracy_score,
+    auc,
+    classification_report,
+    confusion_matrix,
+    f1_score,
+    precision_score,
+    recall_score,
+    roc_auc_score,
+    roc_curve,
```

Comment:
Were all of these imports used in this file? If not, consider removing ones which were not used.